rplugin/python3/gdb/app.py

```diff
@@ -57,6 +58,8 @@ def __init__(self, common, backendStr, proxyCmd, clientCmd):
         # Initialize the parser
         self.parser = self.backend["initParser"](common, self.cursor, self.win)
 
+        self.debugger = Debugger(common, self.parser, self.client)
```

Comment:
The linter complained on too many fields in the class App. So if you could completely transfer the parser and the client into the new class, that'd be awesome. Would it be possible to initialize and even conceal them there?